packages/smart-contracts/src/lib/artifacts/ERC20FeeProxy/index.ts

```diff
@@ -45,6 +45,10 @@ export const erc20FeeProxyArtifact = new ContractArtifact<ERC20FeeProxy>(
           address: '0x0DfbEe143b42B41eFC5A6F87bFD1fFC78c2f0aC9',
           creationBlockNumber: 18326896
         },
+        bsctest: {
```

Comment:
You added twice the same address for two different versions. you should add the address only for the relevant version.

Reply:
thank you, done by keeping it only in the latest version.